qiskit/compiler/sequence.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Mapping a scheduled ``QuantumCircuit`` to a pulse ``Schedule``.
+"""
+from typing import List, Optional, Union
+
+from qiskit.circuit.quantumcircuit import QuantumCircuit
+from qiskit.exceptions import QiskitError
+from qiskit.providers import BaseBackend
+from qiskit.pulse import InstructionScheduleMap, Schedule
+from qiskit.scheduler import ScheduleConfig
+from qiskit.scheduler.methods import sequence as _sequence
+
+
+def sequence(scheduled_circuits: Union[QuantumCircuit, List[QuantumCircuit]],
+             backend: Optional[BaseBackend] = None,
+             inst_map: Optional[InstructionScheduleMap] = None,
+             meas_map: Optional[List[List[int]]] = None) -> Union[Schedule, List[Schedule]]:
```

Comment:
@lcapelluto I'd like to remove `inst_map` and `meas_map` for now because we have not yet ready for the path (custom inst_map for pulse simulation), and addition is easier than removal in the future. For example, we have not yet fully discussed how to help for pulse simulator users to construct an `instruction_durations` compatible with an `inst_map`. Removing `inst_map` and `meas_map` means making backend mandatory. What do you think? Can we remove them for now? Or should we accept them?

Reply:
This method doesn't take or use instruction durations, so why would you have to change this?

Reply:
`sequence` accepts schedule circuits, i.e. `sequence` is always called after calling `transpile`. Some users may use their own `instruction_durations` when transpiling. Should they supply `inst_map` instead of `instruction_durations` when sequencing? This is the case in my mind.
If we could provide `InstructionDurations.from_inst_map()`, they could manage only `inst_map` and make `instruction_durations` on demand for `transpile` using the function. 
I think we'd better to discuss what are appropriate arguments for `sequence` (they might not be `inst_map` and `meas_map`) and what is an easy way for users to prepare the arguments. So for now, I think it would make sense for `sequence` to accept only (pulse-enabled) `backend`. 
Note: We're not talking about `schedule` we already have. We're talking about `sequence` we're newly adding in this PR.

Reply:
OK. Thank you for your explanation. I understand the necessity of `inst_map` and `meas_map` here.